docs/DeveloperGuide.md

```diff
@@ -306,6 +308,26 @@ deleted
 
 ***
 
+### 3.7. Items Add
+![itemsAdd Diagram](https://user-images.githubusercontent.com/77385307/114296774-a7e32d80-9adf-11eb-9def-9fe89db8f5fc.png)
+The `items add` feature is designed to allow users to add items to the system, including the item name, price and stock
+of each item. As seen from the sequence diagram above (note that some trivial details are removed from the diagram), 
+when an item is added into the system:
+1. User executes the `itemsAddcommand` by typing `items add <item_name>`.
+2. easyLog invokes `itemsParser` and calls the `promptAndProcessItemPriceAndStock(String)`method to ask user the
+   `itemPrice` and `itemStock` of this item.
+3. easyLog will keep asking user the `itemPrice` and `itemStock` by calling 
+   `itemPriceInStringToBigDecimalFormat(String)`and `itemStockInStringToIntegerFormat(String)` methods, until the 
+   `itemPrice` and `itemStock` given by the user are both valid.
+4. Then, `itemsParser` creates a new `Item` object, which would be returned to `itemsAddCommand`.
+5. easyLog then invokes `itemsManager` and calls the `addItem(Item)` method to add this item into the system.
+6. After the item is added successfully, `UI` sends back the confirmation message to user by calling `showAddItem(Item)`
+   method.
+   
+[Return to Top](#1-introduction)
+
+***
+
```

Comment:
can shift this part to before orders add so that i won't takeover your code when I rearrange the implementation section